lib/tasks/assets.rake

```diff
@@ -30,9 +30,8 @@ namespace :assets do # rubocop:disable Metrics/BlockLength
       ENV['NODE_ENV'] = 'test'
       ENV['RAILS_ENV'] = 'test'
       Rake::Task['assets:precompile'].invoke
+      Rake::Task["webpack:compile"].invoke
     end
-
-    Rake::Task["assets:precompile"].enhance(%w[webpack:compile])
```

Comment:
wouldn't this disable `webpack:compile` when not using the `test` task

Reply:
the "test task" was created to update assets specifically in local environment and it was added by me because I was too lazy to add `RAILS_ENV=test` before the task. Now it's absolutely superfluous and I've decided to remove it because it's only confusing people.

Reply:
This is not removing the test task but stops running `webpack:compile` when calling `assets:precompile`. `webpack:compile` will run only when calling `assets:precompile:test`. Somehow I can't understand this.

Reply:
Forget about `assets:precompile:test`, only `assets:precompile´ matters. It compiles the asset pipeline (assets) and then webpack (typescript and React).

```rb
Rake::Task["assets:precompile"].enhance do
  Rake::Task["webpack:compile"].invoke
end
```

Reply:
Good, that was my point as I saw that deleted and the test task available. How it's fine. Or maybe it was fine and I got confused previously.